docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,390 @@
+
+
+#### Tag and publish your container
+Once you are satisfied with the functionality of your container, you can publish the Docker image on to the Docker registry. In our case, we execute :
+
+`docker tag time-server docker-user-name/time-server:part1`
+
+This tags our `time-server` image with the Docker repository `time-server` in your Docker user name `docker-user-name` and provides an optional tag `part1`.
+
+Once we tag our image, we have to publish (synonmous with github push) to the Docker registry so that Marathon would be able to discover this in future using an url. We achieve this by executing the command :
+
+`docker push docker-user-name/time-server:part1`
+
+Now that we have the container ready, in the next section we will see how to create a package!
+
+### Step 3 : Creating a DC/OS Package
+In order to create a package, you need to have forked the [Universe repo](https://github.com/mesosphere/universe) and then cloned it so that it is available in your terminal. Once you do this, create a directory named time-server under the repo/package/T directory (as our package name starts with the letter "t"). Inside this directory, if there is already another package with a name of your choice, you have to name your package differently. We create all the required files in this package.
+
+After you read this step, if you need further examples, you can refer to the [repo/packages/H/hello-world](repo/packages/H/hello-world) package.
+
+Each package has its own directory, with one subdirectory for each package revision. Each package revision directory contains the set of files necessary to create a consumable package that can be used by a DC/OS Cluster to install the package. For example, our package would look like this:
+
+```
+└── repo/package/T/time-server
+    ├── 0
+    │   ├── config.json
+    │   ├── resource.json
+    │   ├── marathon.json.mustache
+    │   └── package.json
+    └── ...
+```
+
+In our case, since this is the first version of our time-server, we will create the above directory structure with only one revision (with number 0) and create the required empty files. As the versions of your package grow, this number increments by one unit.
+
+***Tip : When reading the schema json files, look for `required` json field to understand what fields are mandatory***
+
+#### config.json
+As the name says, this file is used for any configuration purposes. This is how our `config.json` would look like:
```

Comment:
Reword this as follows?

As the name says, this file specifies how our package can be configured. This is how our `config.json` should look: